app/models/foreman_virt_who_configure/output_generator.rb

```diff
@@ -105,6 +106,7 @@ def virt_who_output(format = nil)
 rhsm_username=#{service_user_username}
 rhsm_encrypted_password=$user_password
 rhsm_prefix=/rhsm
+#{kubeconfig}
```

Comment:
this will need to be at the end of previous line, we'd always add \n otherwise, so it should be `rhsm_prefix=/rhsm#{kubeconfig}`... even though it looks ugly.